helix-common/src/main/java/org/apache/helix/HelixConfigStringProcessUtil.java

```diff
@@ -0,0 +1,69 @@
+package org.apache.helix;
```

Comment:
Have we considered putting this in `org.apache.helix.util` since this is a util class?

Reply:
Updated.